State-of-Rhode-Island-eCMS/ecms_profile
# Main menu role access

## Summary / Approach

- Installs modules:
  - drupal/simple_menu_permissions
  - menu_admin_per_menu
- Enables them by default, as required dependecies of ecms_base with update hook `ecms_base_update_9094`
- Updates `user.role.content_publisher.yml` with limited permissions to edit menu links from Node Edit UI, but not from the Structure > Menus Admin area
## Testing Instruction(s)

Logged in as a Content Publisher, 
verify you are able to add a menu link to the main navigation menu from the Node edit UI
verify you are able to update and save edited menu link 
verify you are NOT able to use the Structure > Menus admin area

## Metadata

| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? |Y
| Documentation reflects changes? |N
| `CHANGELOG` reflects changes? |Y
| Unit/Functional tests cover changes? |N
| Did you perform browser testing? |Y
| Did you provide detail in the summary on how and where to test this branch? |Y
| Risk level |L
| Relevant links | [RIGA-346](https://thinkoomph.jira.com/browse/RIGA-346)
